src/presets/netlify/preset.ts

```diff
@@ -22,6 +22,9 @@ const netlify = defineNitroPreset(
       dir: "{{ rootDir }}/.netlify/functions-internal",
       publicDir: "{{ rootDir }}/dist/{{ baseURL }}",
     },
+    prerender: {
+      autoSubfolderIndex: false,
+    },
```

Comment:
Could you please add a comment here explaining why we set this? You can use the links I just shared in a comment.

Reply:
Comments added.